yt/frontends/amrvac/fields.py

```diff
@@ -128,12 +120,12 @@ def _setup_velocity_fields(self, idust=None):
 
     def _setup_dust_fields(self):
         idust = 1
+        imax = self.__class__.MAXN_DUST_SPECIES
         while ("amrvac", "rhod%d" % idust) in self.field_list:
-            if idust > MAXN_DUST_SPECIES:
+            if idust > imax:
                 mylog.error(
                     "Only the first %d dust species are currently read by yt. "
-                    "If you read this, please consider issuing a ticket. "
-                    % MAXN_DUST_SPECIES
+                    "If you read this, please consider issuing a ticket. " % imax
```

Comment:
Would it be possible to change `% imax` to `, imax` as in #2820?

Reply:
sure thing

Reply:
done !